.circleci/config.yml

```diff
@@ -288,8 +289,8 @@ jobs:
           name: Zip file size sanity check
           command: |
             ls -lh /tmp/workspace/planet4-master-theme.zip
-            if [ $(wc -c </tmp/workspace/planet4-master-theme.zip) -ge 9000000 ]; then
-              echo "Zip file size exceeds 9MB, probably something went wrong."
+            if [ $(wc -c </tmp/workspace/planet4-master-theme.zip) -ge 20000000 ]; then
```

Comment:
Is this something we should optimize in the future?
Is there any risk with such a big file file?